v1/pipeline/steps/custom/step-group-node.yaml

```diff
@@ -0,0 +1,47 @@
+title: StepGroupNode
+type: object
+required:
+  - spec
+  - type
+properties:
+  desc:
+    type: string
+    desc: This is the description for ShellScriptStepNode
```

Comment:
this description is wronng, plus field here should be description isnt?

Reply:
yes, done